src/site/content/en/fast/codelab-prefetching-wiih-quicklink/index.md

```diff
@@ -0,0 +1,190 @@
+
+```bash
+npm install webpack-route-manifest --save-dev
+```
+`config-overrides.js` is a file placed in your project root directory where you can override existing behaviour of the webpack configuration, without having to [eject the project](https://github.com/facebook/create-react-app/blob/master/packages/cra-template/template/README.md#npm-run-eject).
+
+Open `config-overrides.js` for edit and add the `webpack-route-manifest` dependency at the beginning of the file:
+
+```javascript/1/
+const path = require('path');
+const RouteManifest = require('webpack-route-manifest');
+```
+
+Next, add the following code to configure the `webpack-route-manifest` plugin:
+
+```javascript/0-24/
+module.exports = function override(config) {
+  config.resolve = {
+    ...config.resolve,
+    alias: {
+      '@assets': `${path.resolve(__dirname, 'src/assets')}`,
+      '@pages': `${path.resolve(__dirname, 'src/pages')}`,
+      '@components': `${path.resolve(__dirname, 'src/components')}`,
+    },
+  };
+
+  config.plugins.push(
+    new RouteManifest({
+      minify: true,
+      filename: 'rmanifest.json',
+      routes(str) {
+        let out = str.replace('@pages', '').toLowerCase();
+        if (out === '/article') return '/blog/:title';
+        if (out === '/home') return '/';
+        return out;
+      },
+    }),
+  );
+
+  return config;
+};
+```
+
+The new code does the following:
+
+- `config.resolve` declares variables with the internal routes to pages, assets and components.
+- `config.plugins.push()` creates a `RouteManifest` object and passes it the configuration so that the `manifest.json` file can be generated based on the site's routes and chunks.
+
+The `manifest.json` file will be generated and made available at `https://site_url/rmanifest.json`.
+
+## Configure quicklink
+
+At this point you would need to install the `quicklnk` library in your project. For simplicity, we already added it to the project. Here's the command that you would need to run:
+
+```bash
+npm install quicklink
+```
+
+Open `src/components/App/index.js` for edit.
+
+First, import the quicklnk higher order component (HOC):
+
+```javascript/6/
+import React, { lazy, Suspense } from 'react';
+import { Route } from 'react-router-dom';
+
+import Footer from '@components/Footer';
+import Hero from '@components/Hero';
+import style from './index.module.css';
+import { withQuicklink } from 'quicklink/dist/react/hoc.js';
+
+const Home = lazy(() => import(/* webpackChunkName: "home" */ '@pages/Home'));
+const About = lazy(() => import(/* webpackChunkName: "about" */ '@pages/About'));
+const Article = lazy(() => import(/* webpackChunkName: "article" */ '@pages/Article'));
+const Blog = lazy(() => import(/* webpackChunkName: "blog" */ '@pages/Blog'));
+```
+
+Next, create an `options` object to use as an argument when calling `quicklink`:
+
+```javascript/0-2/
+const options = {
+	origins: []
+};
+```
```

Comment:
Please provide a little more context around where this code should be placed (e.g. under the `Blog` declaration?)